pkg/services/quota/quotaimpl/storage/storage.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I dont think we should commit this file?

Reply:
Yeah, someone forgot to add it to `.gitignore`. I have removed it from this PR.

Reply:
Removed.